source/agora/consensus/Fee.d

```diff
@@ -343,6 +365,19 @@ public class FeeManager
                     return so_far;
                 }
             );
+
+            try
+            {
+                auto new_fee = this.accumulated_fees[stake.output.address]
```

Comment:
This will throw an Error if `accumulated_fees` doesn't have this public key. Is there any case where that could happen?

Reply:
Nope, L361 prevents that.

Reply:
Got it. It didn't show in the diff.